pkgs/by-name/gv/gvmd/package.nix

```diff
@@ -0,0 +1,70 @@
+{ lib, stdenv, fetchFromGitHub, cmake, gvm-libs, pkg-config, gnutls, glib
+, doxygen, libbsd, libical, gpgme, util-linux, libselinux, libsepol, libidn2
+, graphviz, icu, p11-kit, pcre2, libxslt, libtasn1, installShellFiles, xmltoman
+, postgresql, }:
+
+stdenv.mkDerivation rec {
+  pname = "gvmd";
+  version = "23.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gvmd";
+    rev = "v${version}";
+    hash = "sha256-Bxq6o98viMMwt2eoiZgPDTYqD/z6jRWYLgH/ZJvkpV4=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config doxygen xmltoman installShellFiles ];
+
+  buildInputs = [
+    libxslt
+    postgresql
+    pcre2
+    libbsd
+    libidn2
+    gvm-libs
+    graphviz
+    gnutls
+    glib
+    libtasn1
+    icu
+    libical
+    gpgme
+    glib
+    libsepol
+    libselinux
+    util-linux
+    p11-kit
+    postgresql
+    gpgme
+  ];
+
+  strictDeps = true;
+
+  prePatch = ''
```

Comment:
Usually we ordem items by phases order. Which is the execution order of hooks. See [phases order](https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases). Errors also happens in sequence when following phases order.

`prePatch` is patch phase.
`strictDeps`, `cmakeFlags` is configure phase.

Reply:
Explain how this is resolved?

Reply:
made the changes, does it satisfy phases order now?

Reply:
It is not right.

`prePatch` is patch phase.
`strictDeps`, `cmakeFlags` is configure.

Reply:
What's the best approach? Should I use `substituteInPlace` in prebuild phase and what about the strictDeps do I remove it entirely?